### PR TITLE
[docs] Update some chain_api_plugin descriptions

### PR DESCRIPTION
**Pull request by deckb**
## Change Description

Update documentation for some chain_api_plugin endpoints

Additionally change `get_code_params::code_as_wasm` to accurately reflect the documentation and deprecation of WAST retrieval. 

## Change Type
**Select *ONE*:**
- [X ] Documentation

- [ ] Stability bug fix

- [ ] Other

- [ ] Other - special case

## Testing Changes
**Select *ANY* that apply:**
- [ ] New Tests

- [ ] Existing Tests

- [ ] Test Framework

- [ ] CI System

- [ ] Other

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [X ] API Changes

## Documentation Additions
- [ X] Documentation Additions

### CODE REVIEW

**Comment by deckb** [comment]
I have run `thorka` and it built correctly.

@b1bart I assumed that we wanted to change the default for `code_as_wasm` to `true` as it is documented and we no longer allow WAST to be returned.

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp [hunk 1]
```diff
@@ -205,7 +205,7 @@ class read_only {
 
    struct get_code_params {
       name account_name;
-      bool code_as_wasm = false;
+      bool code_as_wasm = true;
```
Comment (lparisc) [inline plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp:208]: Can you please make this code change in a separate commit? The current commit also updates the swagger file. We want to keep code changes separate from docs changes to separate concerns and simplify eventual git operations if needed. @b1bart is this change `false` -> `true` ok?

**Review by lparisc — CHANGES_REQUESTED** [review]

File: plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp (hunk 1 above)
Comment (deckb) [reply plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp]: done.

**Review by lparisc — APPROVED** [review]

**Comment by lparisc** [comment]
Replaced by PR #9729 since it's currently impossible to pass all buildkite tests for forked PRs.
